### PR TITLE
Updated urls in source and endpoint

### PR DESCRIPTION
**Pull request by ksp37**

### CODE REVIEW

File: collection/source.csv [hunk 1]
```diff
@@ -289,12 +300,14 @@ brownfield-land,brownfield-land,local-authority-eng:GOS,d46f17f0731ec70de00ade53
...
 brownfield-land,brownfield-land,local-authority-eng:GRA,8a09c267297ef5b1114ec9d887b5ae80dfb8c706043097bbbaed4cc94c893fc7,https://www.gravesham.gov.uk/home/planning-and-building/local-plan/brownfield-land-register,,,,
 brownfield-land,brownfield-land,local-authority-eng:GRE,86ff4b7dd83348a952c733539fc8bce10a6f7f3d31c15705a54858eeedaa3966,,,,,2019-12-14
-brownfield-land,brownfield-land,local-authority-eng:GRE,41e36be4938cd04e20058823a88e612e6ed9414c12d8ddbe6c966d92057efc6e,https://www.royalgreenwich.gov.uk/info/200191/planning_policy_and_strategy/1955/brownfield_land_register,,,,
+brownfield-land,brownfield-land,local-authority-eng:GRE,41e36be4938cd04e20058823a88e612e6ed9414c12d8ddbe6c966d92057efc6e,https://www.royalgreenwich.gov.uk/info/200191/planning_policy_and_strategy/1955/brownfield_land_register,,,,2020-01-01
+brownfield-land,brownfield-land,local-authority-eng:GRE,86ff4b7dd83348a952c733539fc8bce10a6f7f3d31c15705a54858eeedaa3966,https://www.royalgreenwich.gov.uk/info/200191/planning_policy_and_strategy/1955/brownfield_land_register,,,,
```
Comment (bram2000) [inline collection/source.csv:304]: should this row have a start date?

File: collection/source.csv [hunk 2]
```diff
@@ -795,29 +831,32 @@ brownfield-land,brownfield-land,local-authority-eng:WOK,0eb8e8e249606cf5295ae92c
...
-brownfield-land,brownfield-land,local-authority-eng:WYR,05588db376760416f9d698a5ab405f416d84e733f1b932cb899edf3e606748a6,https://www.wyre.gov.uk/info/200317/planning_policy/1194/brownfield_land_register,,,2018-09-01,
-brownfield-land,brownfield-land,local-authority-eng:WYR,f52de3d0d6050f4313917a3cd4e0728a3400a62df501f80f85d3d13ad9afffd8,,,,,2019-11-24
+brownfield-land,brownfield-land,local-authority-eng:WYR,05588db376760416f9d698a5ab405f416d84e733f1b932cb899edf3e606748a6,https://www.wyre.gov.uk/info/200317/planning_policy/1194/brownfield_land_register,,,2018-09-01,2020-10-08
+brownfield-land,brownfield-land,local-authority-eng:WYR,f52de3d0d6050f4313917a3cd4e0728a3400a62df501f80f85d3d13ad9afffd8,,,,2019-11-24,2020-10-08
+brownfield-land,brownfield-land,local-authority-eng:WYR,4f2c2b32a7fb44c2a778354ff3a42688f888f42bd7a344052ea6e494cca0db2a,https://www.wyre.gov.uk/info/200317/planning_policy/1194/brownfield_land_register,,,2020-10-08,
```
Comment (bram2000) [inline collection/source.csv:859]: is that the right start date?

**Review by bram2000 — CHANGES_REQUESTED** [review]

File: collection/endpoint.csv [hunk 3]
```diff
@@ -265,25 +281,27 @@ c618cc6e158b44a78bcab28f07508182b8af75e818e075ac1f4ec50dd6e6c5f1,https://blackbu
 4e64ce2b041bf35fd0368226ff323579ec85b031e8adb6356bbcf364d339f2ed,http://www.testvalley.gov.uk/assets/attach/5653/Test%20Valley%20Borough%20Council%20Brownfield%20Register.csv,,
 4e83e382960083f7a117a41d19cb73b978d8d60e0bd1d0d8a280e183d95e0bcb,https://www.highpeak.gov.uk/media/4893/Brownfield-Land-Register-2019-CSV-Mar20/Excel/hp_brownfield-land.csv.CSV,2020-03-31,
 4ec57eaec0f01f0289defe7a8c22eff455df88ef72ee3ebc754d3accb177fde5,https://www.scarborough.gov.uk/sites/scarborough.gov.uk/files/files/scarborough_brownfieldregister_2017-12-18_rev1.csv,2017-12-18,
-4eea0bf8a68d2619dfa1cfe867c8398ae15725524cf265a9ac55dc0bd4bab47c,http://www.wiltshire.gov.uk/downloads/10793,,
+4eea0bf8a68d2619dfa1cfe867c8398ae15725524cf265a9ac55dc0bd4bab47c,http://www.wiltshire.gov.uk/downloads/10793,,2020-08-20
+4f2c2b32a7fb44c2a778354ff3a42688f888f42bd7a344052ea6e494cca0db2a,http://www.wyre.gov.uk/download/downloads/id/6185/wyre_council_brownfield_register_september_2019.csv,2020-10-08,
```
[the remark this answers is not in the document]
Comment (ksp37) [reply collection/endpoint.csv:285]: I'm not sure...it was from an earlier change I think:
https://github.com/digital-land/brownfield-land-collection/commit/874677ae17532bafa6a36942e428d3e42ac68bac

File: collection/endpoint.csv [hunk 4]
```diff
@@ -617,23 +650,26 @@ bf3e2b4815b9063d23f14828290bc2152734578e4cc7b9c1c3664bc28ba648ce,http://www.woki
...
-c3f3786b320119d81fa0230f54fc05496ed90a8fa57f2672dd51b49c55f63794,http://www.wiltshire.gov.uk/downloads/15681,,
+c3f3786b320119d81fa0230f54fc05496ed90a8fa57f2672dd51b49c55f63794,http://www.wiltshire.gov.uk/downloads/15681,,2020-08-20
 c40c735f8a48b6815bcd3ac46897d00d307b90ec8aeb4b74f11180f23368ba6b,https://www.sedgemoor.gov.uk/media/3722/sedgemoor-brownfieldregister-2018-11-27-rev2/csv/sedgemoor_brownfieldregister_2018-11-27_rev2.csv,2018-11-27,
-c5aad3633bd009dad7f20ff733d79cf7cc4a4a57acd0987ef4dd3fe77a6c3e2a,https://www.teignbridge.gov.uk/media/8707/brownfieldlandregisterteignbridge2019.csv,2020-02-06,
+c50fe953dd436feb4c34837869aa87ee8f271860cd0151770d859f3aa4f6e04a,https://www.southribble.gov.uk/sites/default/files/brownfield-land.csv,2020-10-14,
+c5aad3633bd009dad7f20ff733d79cf7cc4a4a57acd0987ef4dd3fe77a6c3e2a,https://www.teignbridge.gov.uk/media/8707/brownfieldlandregisterteignbridge2019.csv,2020-02-06,2020-08-14
+c618cc6e158b44a78bcab28f07508182b8af75e818e075ac1f4ec50dd6e6c5f1,https://blackburn-darwen.org.uk/wp-content/uploads/brownfield-land.csv,2020-10-05,
```
[the remark this answers is not in the document]
Comment (ksp37) [reply collection/endpoint.csv:670]: https://github.com/digital-land/brownfield-land-collection/commit/ed28d05f80883ce0754a7b9aeae9929e61cc3f0c

File: collection/endpoint.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (bram2000) [reply collection/endpoint.csv:285]: ah yes, good point!

File: collection/source.csv (hunk 2 above)
Comment (ksp37) [reply collection/source.csv:859]: https://github.com/digital-land/brownfield-land-collection/commit/874677ae17532bafa6a36942e428d3e42ac68bac

File: collection/source.csv (hunk 1 above)
Comment (ksp37) [reply collection/source.csv:304]: Ah yes it should...

**Review by bram2000 — APPROVED** [review]

**Pull request merged**